ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/lldp.j2

```diff
@@ -0,0 +1,42 @@
+{% if lldp is arista.avd.defined %}
+
+# LLDP
+
+## LLDP Summary
+
+### LLDP Global Settings
+
+| State | Management Address | Management VRF | Timer | Hold-Time | Re-initialization Timer | Drop Received Tagged Packets |
+| ----- | ------------------ | -------------- | ----- | --------- | ----------------------- | ---------------------------- |
+{%     set row_state = 'Enabled' %}
+{%     if lldp.run is arista.avd.defined(false) %}
+{%         set row_state = 'Disabled' %}
+{%     endif %}
+{%     set row_receive_packet_tagged_drop = 'Disabled' %}
+{%     if lldp.receive_packet_tagged_drop is arista.avd.defined(true) %}
+{%         set row_receive_packet_tagged_drop = 'Enabled' %}
+{%     endif %}
+| {{ row_state }} | {{ lldp.management_address | arista.avd.default('-') }} | {{ lldp.vrf | arista.avd.default('-') }} | {{ lldp.timer | arista.avd.default('30') }} | {{ lldp.holdtime | arista.avd.default('120') }} | {{ lldp.timer_reinitialization | arista.avd.default('2') }} | {{ row_receive_packet_tagged_drop }} |
```

Comment:
avoid setting EOS default values in here. We normally just use `-` as the default value, since nothing specific has been set.
Few exceptions like the global enable and maybe use `default` for the default value for `lldp.vrf` field.
Instead of setting row_state first, you can just change the column title to `Enabled` and use `| {{ lldp.run | arista.avd.default(true) }} |`